Yank/yamlbuild.py

```diff
@@ -725,11 +725,13 @@ def _setup_molecules(self, *args):
             # we update the 'filepath' key also for OpenEye-generated molecules so
             # we don't need to keep track of the molecules we have already generated
             if extension is None or extension == '.smiles' or extension == '.csv':
-                if not utils.is_openeye_installed():
+                if not utils.is_openeye_installed(oetools=('oechem','oeiupac','oequacpac')):
```

Comment:
Fixed with the OEOmega comment you gave broadly. I did not follow the checks of OpenMolTools through, I figured each function had its complete set of checks as headers, I was wrong apparently.